frontend/src/components/print/print-react/components/picasso/Picasso.jsx

```diff
@@ -56,16 +59,28 @@ function Picasso(props) {
         style={styles.page}
         key={periodUri}
       >
-        <Text
-          id={`${props.id}-${period.id}`}
-          bookmark={{
-            title: props.$tc('print.picasso.title', { period: period.description }),
-            fit: true,
-          }}
-          style={styles.h1}
-        >
-          {props.$tc('print.picasso.title', { period: period.description })}
-        </Text>
+        <View style={picassoStyles.titleContainer}>
+          <Text
+            id={`${props.id}-${period.id}`}
+            bookmark={{
+              title: props.$tc('print.picasso.title', { period: period.description }),
+              fit: true,
+            }}
+            style={{ ...styles.h1, ...picassoStyles.title }}
+          >
+            {props.$tc('print.picasso.title', { period: period.description })}
+          </Text>
+          <Text>{period.camp().organizer}</Text>
+          {period.camp().printYSLogoOnPicasso ? (
+            <YSLogo
```

Comment:
There is a z-index issue, the white border of the table below is visible on the logo sometimes